Tests/TestingApps/PaywallsTester/PaywallsTester/Views/UpsellView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LockedView.swift
+//  PaywallsTester
+//
+//  Created by Andrés Boedo on 9/28/23.
+//
+
+import SwiftUI
+
+struct UpsellView: View {
+    var body: some View {
+        VStack {
```

Comment:
This isn't needed right?

Reply:
I planned on adding more things initially, but never got around to it. I can remove this for now